### PR TITLE
correct html semantic for markdown interpreter

### LINKED ISSUE #861: Use correct html semantic for markers instead of <span class="..." .../>
### Describe the feature

Use `<i/>` tag for italic marker, use `<b/>` tag for bold marker and so on ... instead of classNames for a span.

### Why is this feature needed?

To correctly use HTML semantics and potentially improve SEO

### Additional context

https://github.com/helpwave/web/pull/855

### PR DESCRIPTION
**Pull request by DasProffi**
## Which issues does this pull request close?
closes #861 #852

### CODE REVIEW

**Comment by use-to** [comment]
@MaxSchaefer Linting issues everywhere... This currently blocks the entire development division. Course of action?

**Comment by DasProffi** [comment]
> @MaxSchaefer Linting issues everywhere... This currently blocks the entire development division. Course of action?

This one was just a normal linting error

**Comment by MaxSchaefer** [comment]
> @MaxSchaefer Linting issues everywhere... This currently blocks the entire development division. Course of action?

> This one was just a normal linting error

Exactly. Its not dramatic. Lets go into the review.

File: lib/components/MarkdownInterpreter.tsx [hunk 1]
```diff
@@ -1,151 +1,257 @@
...
+    case 'bold':
+      return <b>{node.children.map((value, index) => <ASTNodeInterpreter key={index} node={value}/>)}</b>
+    case 'italic':
+      return <i>{node.children.map((value, index) => <ASTNodeInterpreter key={index} node={value}/>)}</i>
+    case 'underline':
+      return (
+        <span className={tw('underline')}>{node.children.map((value, index) => (
+          <ASTNodeInterpreter key={index}
+                              node={value}/>
+        ))}</span>
```
Comment (MaxSchaefer) [inline lib/components/MarkdownInterpreter.tsx:45]: A particular reason why we are not using `<u/>` here?

Comment (DasProffi) [reply lib/components/MarkdownInterpreter.tsx:45]: no, I will change it

**Review by MaxSchaefer — APPROVED** [review]
Overkill 🫡

**Pull request merged**